src/Symfony/Component/Ldap/Adapter/AdapterInterface.php

```diff
@@ -44,11 +40,7 @@ public function getEntryManager();
     /**
      * Escape a string for use in an LDAP filter or DN.
      *
-     * @param string $subject
-     * @param string $ignore
-     * @param int    $flags
-     *
      * @return string
      */
-    public function escape($subject, $ignore = '', $flags = 0);
+    public function escape(string $subject, ?string $ignore = '', int $flags = 0);
```

Comment:
Are we sure we want to support `null` here for `$ignore`? There is a test doing that, but the current phpdoc does not document it. Passing an empty string is enough, right? `ldap_escape` does not seem to support `null` explicitly.

Reply:
I agree. This is fixed.

Reply:
Should  be changed everywhere.

Reply:
done